### DISCUSSION TITLE
[External cluster] dashboard, web terminal not available

### DISCUSSION THREAD

**Discussion by lukasz-bielinski** [kubermatic/kubermatic#12316] [General]
Hi, 

I want to ask you, if there is any way to enable web terminal and kubernetes-dashboard on external cluster. I checked this on imported kubeconfig and also on created external eks. I enabled oidc in KubermaticConfiguration

![admin-panel](https://github.com/kubermatic/kubermatic/assets/13658573/007caafe-5e73-48d6-b7d0-6320791404a5)
![imported-eks](https://github.com/kubermatic/kubermatic/assets/13658573/195edc71-b438-4af7-bb19-f5cefccc73af)

**Comment by embik** [comment]
Hi @lukasz-bielinski sorry for the late reply, but the web terminal feature is not available for external clusters as it relies on KKP being able to configure certain options on the cluster's apiserver, something that managed Kubernetes offerings usually don't allow.
